README.md

```diff
@@ -88,6 +88,21 @@ Available `options`:
 }
 ```
 
+### `await seed.putProperty(key, value)`
+
+Add a new Seedbee metadata value.
+
+``` js
+const key = SeedBee.ALLOWED_PEERS_METADATA_KEY
+const value = '*'
+await seed.putProperty(key, value)
```

Comment:
Does the '*' approach still work with the new valueEncoding of an array of buffers?

Reply:
nope, sorry will update readme